### PR TITLE
Transupdate

### PR DESCRIPTION
**Pull request by agaida**

### CODE REVIEW

File: lxqt-transupdate [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#=============================================================================
+
+# lxqt-transupdate
+# Update LXQt translation files.
+
+# .translation-update should be in the repository root and contain
+# the path(es) to the translations as in:
+# translations='./src'
+
+LANG=C
```
[the remark this answers is not in the document]
Comment (agaida) [reply lxqt-transupdate]: Thats simple - if one change something in the code (adding lines etc) - just type `lxqt-transupdate` and be done with. I asked for a tool like that for years, nothing happend, so i wrote the script myself.

Another adventage is that i only need lupdate, no other installed dependencies. And to be honest - i don't like the cmake approach - especially not with the now standard out-of-tree builds.

[the remark this answers is not in the document]
Comment (palinek) [reply lxqt-transupdate]: With the above "why" I meant: why are you forcing the C locale?

**Comment by palinek** [comment]
> Another adventage is that i only need lupdate, no other installed dependencies.

But the disadvantage is, that now you have the configuration about translations in two places (which can get out of sync easily).

**Comment by agaida** [comment]
rigth - we should drop the cmake things.

File: lxqt-transupdate (hunk 1 above)
[the remark this answers is not in the document]
Comment (agaida) [reply lxqt-transupdate]: you are right, not needed here

**Comment by palinek** [comment]
> rigth - we should drop the cmake things.

You can't get rid of it...as it is needed to create the binary translation files.

**Comment by agaida** [comment]
@palinek - we can get rid of some aspects like the translation update - translation updates have to be outside of the normal cmake process - it isn't the right place to do things - and far more important  - a translation update will change nothing. The only thing it change is the translation base file - so what? It doesn't translate some new strings automagically into the target languages nor it has any effect on strings that are gone - and even in Arch nobody push these changes back to the repo - so we can cut out this step.

**Comment by agaida** [comment]
i need this merged for more tests - if it turns out to be a bad idea it can go away

**Pull request merged**